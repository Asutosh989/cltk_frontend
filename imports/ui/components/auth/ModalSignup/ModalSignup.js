import baseTheme from 'material-ui/styles/baseThemes/lightBaseTheme';
import getMuiTheme from 'material-ui/styles/getMuiTheme';

ModalSignup = React.createClass({

	propTypes: {
		lowered: React.PropTypes.bool,
		closeModal: React.PropTypes.func,
	},

	childContextTypes: {
		muiTheme: React.PropTypes.object.isRequired,
	},

	getChildContext() {
		return { muiTheme: getMuiTheme(baseTheme) };
	},

	render() {
		const lowered = this.props.lowered;

		return (
			<div
				className={`cltk-modal-signup cltk-modal cltk-login-signup
					${((lowered) ? ' lowered' : '')}`}
			>
				<div
					className="close-modal paper-shadow"
					onClick={this.props.closeModal}
				>
					<i className="mdi mdi-close" />
				</div>
				<div className="modal-inner">
					<BlazeToReact blazeTemplate="atForm" state="signUp" />
				</div>
			</div>
		);
	},
});
