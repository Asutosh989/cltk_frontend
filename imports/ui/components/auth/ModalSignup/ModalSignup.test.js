
import React from 'react';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

// component:
import ModalSignup from './ModalSignup';

describe('ModalSignup', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(
				<MuiThemeProvider>
					<ModalSignup isTest />
				</MuiThemeProvider>
			)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
