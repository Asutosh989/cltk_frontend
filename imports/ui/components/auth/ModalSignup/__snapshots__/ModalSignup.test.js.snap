// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalSignup renders correctly 1`] = `
<div
  className="cltk-modal-signup cltk-modal cltk-login-signup
					"
>
  <div
    className="close-modal paper-shadow"
    onClick={undefined}
  >
    <i
      className="mdi mdi-close"
    />
  </div>
  <div
    className="modal-inner"
  >
    
  </div>
</div>
`;
